data/parameters.js

```diff
@@ -1384,69 +1384,45 @@ const params = {
         contact: 'If you have any questions about access or the registration process, please contact ',
         email: 'support@datacommons.io',
       },
-      dataExplorerTableProperties: {
-        tableConfig: {
-          timestamp: '2018-01-12T16:42:07.495Z',
-          type: 'subjects',
-          keyField: 'name',
-          defaultSorted: [{ id: 'name', desc: false }],
-          columns: [
-            {
-              show: true,
-              Header: 'Case.ID',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'name',
-            },
-            {
-              show: true,
-              Header: 'Project',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'project',
-            },
-            {
-              show: true,
-              Header: 'Study',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'study',
-            },
-            {
-              show: true,
-              Header: 'Gender',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'gender',
-            },
-            {
-              show: true,
-              Header: 'Ethnicity',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'ethnicity',
-            },
-            {
-              show: true,
-              Header: 'Race',
-              type: 'string',
-              sortable: true,
-              canChangeShow: true,
-              accessor: 'race',
-            },
-          ],
-        },
-        graphqlQuery: '{ subject { hits { edges { node { name, project, study, gender, ethnicity, race, file_type }}}}}',
-      },
     },
     featureFlags: {
       explorer: true,
     },
+    arrangerConfig: {
+      charts: {
+        project: {
+          chartType: 'count',
+          title: 'Projects',
+        },
+        study: {
+          chartType: 'count',
+          title: 'Studies',
+        },
+        file_type: {
+          chartType: 'count',
+          title: 'File Types',
+        },
+        ethnicity: {
```

Comment:
Patch things so the explorer page still works even if `arrangerConfig` is not defined for a particular project.  Have a default `config` with some default fields (project, study, case, race, whatever) that the code falls back to ...

Reply:
Actually - it looks like you already handle this by defaulting `arrangerConfig` to `{}`  - awesome!
You can ignore the previous comment if that works ok ...